src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerCFTTest.java

```diff
@@ -1334,12 +1339,56 @@ public void should_return_a_200_with_multiple_tasks_in_search_results_when_avail
             .statusCode(HttpStatus.OK.value())
             .body("tasks.size()", is(2)) //Default max results
             .body("tasks.id", hasItems(taskId1, taskId2))
+            .body("tasks.task_state", everyItem(either(is("unassigned")).or(is("assigned"))))
             .body("tasks.permissions.values", everyItem(hasItem("Own")))
             .body("total_records", is(2));
 
         common.cleanUpTask(taskId1, taskId2);
     }
 
+    @Test
+    public void should_return_a_200_with_search_results_when_queried_with_single_task_request_available_tasks_only() {
+
+        String taskType1 = "reviewAdditionalHomeOfficeEvidence";
+        String taskType2 = "reviewAdditionalAppellantEvidence";
+
+        String caseId = given.iCreateACcdCase();
+        List<CamundaTask>  camundaTasks = common.setupTaskAndRetrieveIdsForGivenCaseId(caseId, taskType1);
+        String taskId1 = camundaTasks.get(0).getId();
+
+        camundaTasks = common.setupTaskAndRetrieveIdsForGivenCaseId(caseId, taskType2);
+        String taskId2 = camundaTasks.get(0).getId();
+
+        common.setupCFTOrganisationalWithMultipleRoles(headers);
+
+        // insert taskId1
+        common.insertTaskInCftTaskDb(new TestVariables(caseId, taskId1, "processInstanceId1"),
+            taskType1, headers);
+
+        // insert taskId2
+        common.insertTaskInCftTaskDb(new TestVariables(caseId, taskId2, "processInstanceId1"),
+            taskType2, headers);
+
+        SearchTaskRequest searchTaskRequest = new SearchTaskRequest(asList(
+            new SearchParameterBoolean(AVAILABLE_TASKS_ONLY, SearchOperator.BOOLEAN, true)
+        ));
+
+        Response result = restApiActions.post(
+            ENDPOINT_BEING_TESTED  + "?first_result=0&max_results=10",
+            searchTaskRequest,
+            headers
+        );
+
+        result.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .body("tasks.size()", lessThanOrEqualTo(10))
+            .body("tasks.task_state", everyItem(either(is("unassigned")).or(is("assigned"))))
```

Comment:
I think we can cover both AC's by using hasItems("unassigned", "assigned") and not(hasItems("unconfigured", "completed", "cancelled", "terminated")

Reply:
We are verifying to have either 'unassigned' or 'assigned' , which means no other values are expected.